package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "hansken-js",
-    "version": "0.3.0",
+    "version": "0.3.1",
```

New functionality ➛ bump minor version (https://semver.org/)
```suggestion
    "version": "0.4.0",
```